R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -2370,6 +2370,15 @@ test_that("join(), crossJoin() and merge() on a DataFrame", {
   expect_true(any(grepl("BroadcastHashJoin", execution_plan_broadcast)))
 })
 
+test_that("test hint", {
```

Comment:
merge this with the previous test - less test set is preferred

Reply:
The previous test with join broadcast hint is for test "join(), crossJoin() and merge() on a DataFrame". It seems to me that a simple sql hint test doesn't belong to that catalog. I guess it might be better to separate.  What do you think?

Reply:
ok